source/documentation/common/upgrade/proc-Upgrading_hypervisor_preserve_gluster_storage.adoc

```diff
@@ -344,7 +344,7 @@ If there are any failures during self-hosted engine deployment, find the problem
 Make sure that the installation does not format the other disks, as bricks are created on top of those disks.
 ====
 
-.. If multipath configuration is not available on the newly installed host, blacklist the Gluster devices. The inventory file is already created in the first host as part of the step _Blacklist the devices used for Gluster bricks_.
+.. If multipath configuration is not available on the newly installed host, disable the Gluster devices. The inventory file is already created in the first host as part of the step _Disable the devices used for Gluster bricks_.
```

Comment:
> step _Disable the devices used for Gluster bricks_

This looks odd to me. I'm not sure we use italics and refer to other steps in a procedure this way. I will let it go because it is outside the scope of this PR but I want to draw it to your attention.

Reply:
I see that this whole module is a mess. :-( 

Not your fault. If Gluster were not deprecated, I would create a task to fix the module.